setup/ci/jenkins.md

```diff
@@ -37,6 +39,10 @@ echo $PASSWORD | hal config ci jenkins master add my-jenkins-master \
                # in your .bash_history
 ```
 
+> *Note*: If you use the [GitHub OAuth plugin](https://github.com/settings/tokens)
```

Comment:
That link is to retrieve a token, not the plugin

Reply:
I think you want this: https://wiki.jenkins.io/display/JENKINS/GitHub+OAuth+Plugin